### ISSUE TITLE
How much did you clip the gradient?

### ISSUE THREAD

**Issue by mks0601** [matterport/Mask_RCNN#146]
Hi, thanks for sharing your code.
You mentioned that you clipped the gradient in REAME.md
How much did you clip the gradient? I cannot find that in your code

**Comment by waleedka** [comment]
It's in `model.py` in `compile()`:

```
optimizer = keras.optimizers.SGD(lr=learning_rate, momentum=momentum,
                                 clipnorm=5.0)
```
